src/test/java/com/epam/lab/payments/MainRestControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.lab.payments;
+
+
+import com.epam.lab.payments.dao.UserRepository;
+import com.epam.lab.payments.domain.UserEntity;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.nio.charset.Charset;
+import java.util.Arrays;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = Payments.class)
+@WebAppConfiguration
+public class MainRestControllerTest {
+
```

Comment:
Why you need empty lines after class name or method name? If u explain me - ok, else - delete.

Reply:
Usual practice - test controlelrs without mockMvc. It's useless. Just call method from controller.

Reply:
Checkstyle recommend to divide methods